DEPENDENCIES/ML-HElib/src/helayers/hebase/AlwaysAssert.cpp

```diff
@@ -22,46 +22,33 @@
 * SOFTWARE.
 */
 
-#include "PTile.h"
+#include "AlwaysAssert.h"
+#include <cstdlib>
+#include <iostream>
+#include <string>
 
 using namespace std;
```

Comment:
For code clarity and prevent ambiguity, I would recommend I would avoid `using namespace xxx` and explicitly call the functions using its fully qualified name, e.g `std::cout`.

Reply:
github issue will be created to revisit Namespace directive style coices.